src/app/features/edit-nonce-drawer/components/edit-nonce-form.tsx

```diff
@@ -11,15 +11,7 @@ function EditNonceFormFallback() {
         <EditNonceField />
       </Stack>
       <Stack isInline>
-        <Button
-          _hover={{
-            boxShadow: 'none',
-          }}
-          boxShadow="none"
```

Comment:
I noticed when looking through some drawers for similar code, that we override button styles. Unless absolutely necessary we shouldn't do this, either changing at source, or not at all.

Reply:
I figured this was the case. I don't follow though. 

> why have the customizations available if you shouldn't use them?

Consistency. Either we change all the buttons, or we don't change any—anything else introduces subtle inconsistencies that add up.

For example, if our new designer joins and uses a slightly different button style to Fer, do we only use that new button going forward? Now we have 3 slightly different buttons in the app.

I wasn't really aware of these button components and have always defaulted to using `<Button mode="..." />`.

When Eduard joins we can pick up the conversation about how we'll eventually migrate to a different library. Even before we have the chance to do that, agreeing on how to handle styles going forward will make it easier when we get to it.

Reply:
My thought is that our `Button` component from the UI library should be used by the app > customized > reused in the app ...like our `PrimaryButton` ...rather than customizing it at the `Button` level. Not sure if that thinking is correct.

Reply:
This'd be a good pattern to follow for sure. We could try this for the buttons? If we do, I'd use dep cruiser/eslint to enforce no importing of Button (bar the one place where it's configured & reexported)